htdocs/comm/mailing/class/html.formadvtargetemailing.class.php

```diff
@@ -171,6 +171,9 @@ public function multiselectselectSalesRepresentatives($htmlname, $selected_array
 		if (!empty($conf->global->USER_HIDE_INACTIVE_IN_COMBOBOX)) {
 			$sql_usr .= " AND u2.statut<>0 ";
 		}
+		if (!empty($conf->global->USER_HIDE_EXTERNAL_IN_COMBOBOX)) {
+			$sql_usr .= " AND u2.employee<>0 ";
```

Comment:
May be you mean  USER_HIDE_NONEMPLOYEE_IN_COMBOBOX  ?  (external term is used for external users so fk_soc > 0).
If you really mean hide external user, condition should be AND u2.fk_soc IS NULL

Reply:
Indeed ! In the end, I have added the two options for non employee and external user if it's fine for you @eldy